WordPress/src/main/res/values/styles_toolbar.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <style name="WordPress.ToolBar" parent="Widget.MaterialComponents.Toolbar.Surface">
```

Comment:
Suggestion (💡): What about further enhancing the structure of these file with extract block comments? For example, if you apply my above suggestions, within this file we can end up with a structure like the below:

```
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <!-- Login -->

    ...

    <!-- App -->

    ...

    <!-- ZenDesk -->

    ...

    <!-- TextAppearance -->

    ...

</resources>
```

PS: Same apply to all such files.

Reply:
In 6ad5f16